reasoner/controller/ConcludableController.java

```diff
@@ -180,20 +180,22 @@ protected Driver<ConclusionController.Explain> registerConclusionController(Rule
             PROCESSOR extends AbstractProcessor<INPUT, OUTPUT, REQ, PROCESSOR>
             > extends AbstractProcessor<INPUT, OUTPUT, REQ, PROCESSOR> {
 
-        private final ConceptMap bounds;
+        final Concludable concludable;
+        final ConceptMap bounds;
         private final Set<Variable.Retrievable> unboundVars;  // TODO: Can just use a boolean to indicate if fully bound
         private final Map<Conclusion, Set<Unifier>> conclusionUnifiers;
         private final Set<Identifier> requestedConnections;
         final java.util.function.Supplier<FunctionalIterator<ConceptMap>> traversalSuppplier;
 
         Processor(Driver<PROCESSOR> driver,
                   Driver<? extends AbstractController<?, INPUT, OUTPUT, REQ, PROCESSOR, ?>> controller,
-                  Context context, ConceptMap bounds,
+                  Concludable concludable, Context context, ConceptMap bounds,
```

Comment:
Concludable should come after context

Reply:
It comes before in the controller constructor. Should I change both?